src/index.jsx

```diff
@@ -35,4 +34,4 @@ App.subscribe(APP_ERROR, (error) => {
   ReactDOM.render(<ErrorPage message={error.message} />, document.getElementById('root'));
 });
 
-App.initialize({ messages: [appMessages, headerMessages], loggingService: NewRelicLoggingService });
+App.initialize({ messages: [appMessages, headerMessages] });
```

Comment:
This change is just opportunistic, as loggingService is now defaulted to NewRelicLoggingService in version 2.2.0.  Arguably should have been in a different PR.

Reply:
Same PR was fine. Just wanted to ensure you wanted it.  Thanks.

Reply:
Same PR was fine. Just wanted to ensure you wanted it. Thanks.